docs/packages/Rule_Tester.mdx

```diff
@@ -281,3 +303,41 @@ import ValidTestCase from '!!raw-loader!../../packages/rule-tester/src/types/Val
 import InvalidTestCase from '!!raw-loader!../../packages/rule-tester/src/types/InvalidTestCase.ts';
 
 <CodeBlock language="ts">{InvalidTestCase}</CodeBlock>
+
+## Static Properties
+
+Each of the following properties may be assigned to as static members of the `RuleTester` class.
+
+For example, to assign `afterAll`:
+
+```ts
+import { RuleTester } from '@typescript-eslint/rule-tester';
+
+RuleTester.afterAll = () => {
+  // ...
+};
+```
+
+### `afterAll`
+
+Runs after all the tests in this file have completed.
+
+### `describe`
+
+Creates a test grouping.
+
+### `describeSkip`
+
+Skips running the tests inside this `describe` for the current file.
+
+### `it`
+
+Creates a test closure.
+
+### `itOnly`
+
+Only runs this test in the current file.
```

Comment:
The existing wording admits at least two interpretations
1. "Only runs _this_ test (in the current file)." - the intended interpretation
2. "Only runs this test in the _current_ file." - a wrong interpretation that makes it sound like a test can be run in multiple files otherwise, whatever that might mean. (this is my initial parse of this sentence)

```suggestion
Skips all other tests in the current file.
```